corehq/apps/app_manager/static/app_manager/js/forms/form_view.js

```
@@ -64,18 +64,35 @@
         };
+        self.unusedVariable = 42;
 
         self.allowed = ko.computed(function () {
-            return !self.formFilter() || !self.caseReferenceNotAllowed() && !self.usercaseReferenceNotAllowed();
+            return !self.form_filter() || !self.caseReferenceNotAllowed() && !self.usercaseReferenceNotAllowed();
         });
 
         return self;
     }
 
+    let data = 5;
```

ESLint

[error] 'data' is assigned a value but never used.

(no-unused-vars)